lib/util/info.js

```diff
@@ -29,6 +30,7 @@ function Info(property, attribute, mask, space) {
   mark(this, 'commaSeparated', check(mask, types.commaSeparated))
   mark(this, 'spaceSeparated', check(mask, types.spaceSeparated))
   mark(this, 'commaOrSpaceSeparated', check(mask, types.commaOrSpaceSeparated))
+  mark(this, 'unspecified', isUnspecifiedProperty(property, attribute, space))
```

Comment:
Would it make sense to have to constructors? One that’s used by the schema‘s, one for [`find` as a fallback](https://github.com/wooorm/property-information/blob/911285c79dbf8f8eb842849536324ffa196f8c88/find.js#L31)?

Reply:
Sorry. Well, `Info` is a class / constructor. We could rename it to `DefinedInfo`.
And create a new class for `UndefinedInfo`, and use that for `find` results that are not defined.
(or other names)

Reply:
Ah, you can want to classify it with the help of constructors. I like the idea. I will update the PR tomorrow 👍

Reply:
I think it would be good for performance to have all “classified” properties inherit from one thing.
And on `.find()`, when there’s no result, construct something specific that has everything set to `false`, plus this new field!

Reply:
e.g
```
Info (attribute, property, space ...) -> DefinedInfo
     						          -> UndefinedInfo (unspecified)
```